srv/salt/ceph/configuration/files/ceph.conf.j2

```diff
@@ -18,6 +18,13 @@ ms_bind_msgr2 = false
 mon_health_preluminous_compat = true
 mon health preluminous compat warning = false
 
+{% set ipv6=salt['public.ipv6']() %}
+{% if ipv6 %}
+ms bind ipv4 = {{ salt['public.ipv4']() | lower }}
```

Comment:
I assume this means ceph can now run with both ipv4 and ipv6? Otherwise I'm confused why ipv6 is hardcoded and ipv4 is queried.

Reply:
@jan--f Yes, it was for future compatibility.  The validation prevents IPv4 and IPv6 today, but Ceph upstream is planning on supporting both.  When that happens, only the validation will need correcting.  If an admin uses both IPv4 and IPv6 networks, then the ceph.conf will be correct.

As far as hardcoding the IPv6 setting, it will always match the conditional.  So, no need to query a second time.